src/Illuminate/Foundation/Console/stubs/echo-bootstrap-js.stub

```diff
@@ -4,4 +4,4 @@
  * allow your team to quickly build robust real-time web applications.
  */
 
-import './echo';
+import './echo.js';
```

Comment:
Removing this for now. @yourchocomate please send in a separate PR with explanation if you need this.

```suggestion
```

Reply:
Created a PR  #50520 regarding the issue I mentioned , you can check this out. Thanks